identity_account_storage/src/storage/traits.rs

```diff
@@ -151,17 +149,11 @@ pub trait Storage: storage_sub_trait::StorageSendSyncMaybe + Debug {
     private_key: &KeyLocation,
   ) -> Result<Vec<u8>>;
 
-  /// Returns the chain state of the identity specified by `did`.
-  async fn chain_state_get(&self, did: &IotaDID) -> Result<Option<ChainState>>;
+  /// Stores an arbitrary value for the identity specified by `did`.
+  async fn blob_set(&self, did: &IotaDID, value: &[u8]) -> Result<()>;
```

Comment:
I think this should take `value` as an owned `Vec<u8>` to prevent needing to copy the entire serialized blob.